admin/Dockerfile

```diff
@@ -10,11 +10,11 @@ RUN cd libs && \
     cd ../admin && \
     npm ci && \
     npm run build:admin && \
-    rm -rf node_modules && \
-    npm i typescript@~4.1.6 express@^4.17.1 helmet@^4.6.0
+    rm -rf node_modules
 
 # Deployment container
-FROM registry.access.redhat.com/ubi8/ubi-micro
+# FROM registry.access.redhat.com/ubi8/ubi-micro
+FROM caddy:2
```

Comment:
Derek will want to try to continue to use ubi-micro as the base, but then this means installing Caddy manually, you might want to leave this up to Derek to sort out.

Reply:
@basilv Thanks, agreed!  Ian and I have decided to change the base later.  For now, this is bang on.  :)